experiments/enableTTL/README.md

```diff
@@ -0,0 +1,188 @@
+
+### With `enableTTL`
+
+With `enableTTL` set and a corresponding .ttl file next to the cached resource in question, you will see log entries that match the following format.
+
+When the current time is BEFORE the last modified timestamp of a cached resource's .ttl file, it is considered VALID:
+
+```
+[..] cache file has not expired: /docroot/content/dispatchertester/us/en.html
+[..] cache-action for [/content/dispatchertester/us/en.html]: DELIVER
+```
+
+When the current time is AFTER the last modified timestamp of a cached resource's .ttl file, it is considered STALE:
+
+```
+[..] cache file has expired: /docroot/content/dispatchertester/us/en.html
+[..] cache-action for [/content/dispatchertester/us/en.html]: CREATE
+```
+
+### Without `enableTTL`
+
+When `enableTTL` is not set, or a .ttl file is missing for the resource in question, the logs will read differently. When the cached resource is newer than it's nearest .stat file, the cached file is considered VALID:
+
+```
+[..] cache file is newer than lastflush -> use cache [/docroot/content/dispatchertester/us/en.html]
+[..] cache-action for [/content/dispatchertester/us/en.html]: DELIVER
+```
+
+When it's older than it's nearest .stat file, the file is considered STALE:
+
+```
+[..] cache file is older than lastflush -> flush [/docroot/content/dispatchertester/us/en.html]
+[..] cache-action for [/content/dispatchertester/us/en.html]: CREATE
+```
+
+Ensure you are seeing the "cache file has not expired"/"cache file has expired" format in your local dispatcher logs (on macOS: `/private/var/log/apache2/dispatcher.log`) before proceeding.
+
+## Different TTL values
+
+Depending on your use case, you may decide that a TTL of 5 minutes for .html content is reasonable, but cached .js and .css files should live in the cache for 24 hours (perhaps due to your use of [Versioned Clientlibs](https://adobe-consulting-services.github.io/acs-aem-commons/features/versioned-clientlibs/index.html)).
+
+To achieve this we will create another `DispatcherMaxAgeHeaderFilter` configuration, which will control the TTL on our app's clientlibs.
+
+Navigate to CRX/de on your publish instance, and open `/apps/dispatchertester/config`: [localhost:4503/crx/de/index.jsp#/apps/dispatchertester/config](http://localhost:4503/crx/de/index.jsp#/apps/dispatchertester/config)
+
+Create a copy of the existing `DispatcherMaxAgeHeaderFilter` file and paste it next to the original. Rename it's suffix from `set-1` to `set-2`. Make sure to save your changes.
+
+Open the properties of your new node, and set the `filter.pattern` entry to:
+
+```
+/etc.clientlibs/dispatchertester/clientlibs/(.*)
+```
+
+Additionally, set `max.age` to `86400` (60 seconds in a minute * 60 minutes in an hour * 24 hours in a day = 86400 seconds in a day). This means that all cached items that exist under this path will be able to live in the cache for 24 hours.
```

Comment:
You probably don't need to be as verbose here defining that 86400 = 24 hours.

Reply:
I thought 86400 seemed like an oddly specific number of seconds, so I thought it might help to break it down.